<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplashScreen</class>
 <widget class="QMainWindow" name="SplashScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>554</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>554</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Splash Screen</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius: 10px;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">border-style: solid;
border-width: 2px;
border-color:rgb(61, 64, 70);
background-color: rgb(32, 33, 36);
border-radius: 10px;
background-color: rgb(32, 33, 36);
border-radius: 10px;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="icon_frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 33, 36);
border-radius: 10px;

border-width: 0px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Microsoft PhagsPa</family>
           <pointsize>13</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(214, 214, 214);</string>
         </property>
         <property name="text">
          <string>Maths Visualisation</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>300</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius:10px;
padding-right: 20px;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>images/icon.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="indent">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="buttons_frame">
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>450</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
border-top:    1px solid  rgb(61, 64, 70);
border-right:  0px solid;
border-bottom: 1px solid  rgb(61, 64, 70);
border-left:  0px solid;
background-color: rgb(32, 33, 36);
border-radius:0px
}

</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="new_button">
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>75</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string extracomment="New.."/>
         </property>
         <property name="toolTipDuration">
          <number>2</number>
         </property>
         <property name="statusTip">
          <string extracomment="New.."/>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover {
	background-color: rgb(56, 57, 61);
}
QPushButton{
border-width: 0px;}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/new image.png</normaloff>images/new image.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>85</width>
           <height>100</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exit_button">
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>75</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string extracomment="Exit"/>
         </property>
         <property name="toolTipDuration">
          <number>2</number>
         </property>
         <property name="statusTip">
          <string extracomment="Exit"/>
         </property>
         <property name="whatsThis">
          <string extracomment="Exit"/>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover {
	background-color: rgb(56, 57, 61);
border-width: 0px;
}
QPushButton{
border-width: 0px;}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/outline_exit_to_app_white_48dp.png</normaloff>images/outline_exit_to_app_white_48dp.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>85</width>
           <height>100</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>exit_button</zorder>
      <zorder>new_button</zorder>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="footer_frame">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
border-width: 0px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="version_label">
         <property name="font">
          <font>
           <family>Microsoft PhagsPa</family>
           <pointsize>9</pointsize>
           <italic>false</italic>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(214, 214, 214);</string>
         </property>
         <property name="text">
          <string>Maths Visualisation 1.0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="tooltip">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(214, 214, 214);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
